### DISCUSSION TITLE
Cypress grep returns odd error after tests being run

### DISCUSSION THREAD

**Discussion by Dzanan91** [cypress-io/cypress#25469] [Questions and Help]
Got this error when running tests with tags using cypress/grep : 

**An uncaught error was detected outside of a test:
     TypeError: The following error originated from your test code, not from Cypress.

  > (0 , support_1.registerCypressGrep) is not a function**

support file : 

// "import" with `@ts-ignore`
// @see error 2306 https://github.com/microsoft/TypeScript/blob/3fcd1b51a1e6b16d007b368229af03455c7d5794/src/compiler/diagnosticMessages.json#L1635
// @ts-ignore
import registerCypressGrep from '@cypress/grep'
registerCypressGrep()

config file: 

setupNodeEvents(on, config) {
      require('@cypress/grep/src/plugin')(config);
      return config;
    },

**Comment by sumandeepbkaur** [comment]
Getting same error

**Reply by ENdonga** [reply]
check my comment down, worked for me when i added this `env: { grepFilterSpecs: true, grepOmitFiltered: true },`

**Comment by ENdonga** [comment]
in the e2e support file add the following:
```
const registerCypressGrep = require("@cypress/grep");
registerCypressGrep();
```

in the config file cypress.config.ts add the following:
// cypress.config.js
```
module.exports = defineConfig({
  env: { grepFilterSpecs: true, grepOmitFiltered: true },
  ...
})
```
```
{
  e2e: {
    setupNodeEvents(on, config) {
      require('@bahmutov/cy-grep/src/plugin')(config);
      // IMPORTANT: return the config object
      return config;
    },
  }
}
```
now the error should be gone.
Follow this [link](https://github.com/bahmutov/cy-grep) for more information
